lib/rentlinx/client.rb

```diff
@@ -0,0 +1,32 @@
+require 'httpclient'
+require 'json'
+require 'uri'
+require 'rentlinx/default'
+
+module Rentlinx
+  # Client is the primary interface that users of this gem will utilize.
+  class Client
+    def initialize(username, password, api_url_prefix)
+      @url_prefix = (api_url_prefix + '/').freeze  # Extra slashes are fine
```

Comment:
Yes, your recommendation would make the `@url_prefix` be more accurate. The way we use it however, will correct any issues.:

```
URI.join('http://foo.com/a///', 'bar').to_s
 => "http://foo.com/a/bar" 
```